src/venue/style.css

```
@@ -0,0 +1,242 @@
+.header {
+  position: relative;
+  background: var(--white);
+  padding: 0;
+
+  @media (min-width: 480px) {
+    padding: 3rem var(--side-margin) 3rem;
+  }
+
+  @media (min-width: 1024px) {
+    padding: 8rem var(--side-margin) 2rem;
+  }
+}
+
+.header-map {
+  background: var(--lesslessdarkoffwhite) url('confboxAsset(assets/map.jpg)')
+    top right no-repeat;
+  background-size: cover;
+  padding-top: 85%;
+
+  @media (min-width: 480px) {
+    padding: 0;
+    background-size: contain;
```

```suggestion
    background-size: 50% auto;
    background-position: 100% 50%;
```

I'd argue that this makes better use of the available space on bigger widths.

---

i tested both, prefer jake's choice here